<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Category Layout</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
        QWidget
        {
            color: #fff;
            background-color: #01889C;
            selection-background-color:#3d8ec9;
            selection-color: black;
            background-clip: border;
            border-image: none;
            outline: 0;
        }

        QPushButton
        {
            color: silver;
            background-color: #484846;
            border-width: 2px;
            border-color: #4A4949;
            border-style: solid;
            padding-top: 2px;
            padding-bottom: 2px;
            padding-left: 10px;
            padding-right: 10px;
            border-radius: 4px;
        }

        QPushButton:focus {
            background-color: #302F2F;
            color: white;
        }

        QPushButton:checked{
            background-color: #302F2F;
            border-color: #6A6969;
        }

        QPushButton:hover {
            border: 2px solid #fff;
            color: silver;
        }

        QPushButton:pressed
        {
            background-color: #484846;
        }

    </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_word">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>701</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_a">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image/strawberry.png</normaloff>image/strawberry.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_b">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image/cat.png</normaloff>image/cat.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_c">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image/ball.png</normaloff>image/ball.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_score">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wrong_letters">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
